### ISSUE TITLE
Updated kata-qemu-sev SEV-ES guide

### ISSUE THREAD

**Issue by tylerfanelli** [confidential-containers/confidential-containers#183]
@Xynnn007 @fitzthum @wainersm 

A guide to attesting SEV-ES encrypted images with `kata-qemu-sev` and `simple-kbs` exists here:

https://github.com/confidential-containers/confidential-containers/blob/main/guides/sev.md

Is there any updated guides of attesting images with the current `kbs/attestation-service` and `attestation-agent`?

**Comment by fitzthum** [comment]
The KBS doesn't support SEV(-ES). We have been focusing entirely on SNP there and using the old simple-kbs for -ES. Is SEV(-ES) a big priority for you? It should be feasible to implement for the KBS but it just has not been a priority.

**Comment by tylerfanelli** [comment]
No, SEV-ES isn't a priority. Is there a similar demo available for SEV-SNP?

**Comment by fitzthum** [comment]
The main quickstart guide describes using the KBS. It is not SNP-specific, though. There are probably places where we should expand the docs. Is there something in particular you are looking for information about?

**Comment by tylerfanelli** [comment]
I see no mention of a `kata-qemu-snp` runtime class. I'm assuming this would be different than the `kata-qemu-sev` runtime class due to post attestation. Does this exist yet? Although the `kata-qemu-sev` guide used `simple-kbs`, `kata-qemu-tdx` uses `kbs/attestation-service`, which uses the standard KBS HTTP protocol. I'm hoping a `kata-qemu-snp` runtime class would do the same.

**Comment by tylerfanelli** [comment]
I think [this](https://github.com/kata-containers/kata-containers/blob/f056ffe5effe2a34dd2201ea0a745c9e0c9e2a97/versions.yaml#L113-L116) is what I'm looking for?
